authBackEnd/apps/common/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class TimeStamp(models.Model):
```

Comment:
This should probably be moved to the api models and all other models should subclass from it.

Reply:
The TimeStamp model can be imported and inherited/used in the api models... Done